app/src/main/java/com/example/sharingang/NewItemActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.sharingang
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.example.sharingang.databinding.ActivityNewItemBinding
+
+class NewItemActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityNewItemBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_new_item)
+    }
+
+    fun createNewItem(view: View) {
+        val intent = Intent(baseContext, MainActivity::class.java)
```

Comment:
What is "baseContext"?

Reply:
So you need some kind of `Context` class when starting an Intent, and `baseContext` (shorthand for `this.getBaseContext()`) is one way of getting at that. (I can't honestly claim to understand how contexts work that well, there's like 4 methods to get a context, and this was the one that happened to be used when I looked up how to start an Intent)

That being said, I think using `this` instead would also work, since Activities are also Contexts, and would probably be a lot clearer.

Reply:
Since I'm probably not the only one in the group unfamiliar with Kotlin, I think it might be less confusing to avoid using the shorthands notations (aka use this. instead of just the property), if you don't mind.